spec/features/user_logs_in_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+feature 'User Logs In' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.